app/Console/Commands/GeohubImport.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Enums\UserRole;
+use App\Models\EcTrack;
+use Illuminate\Console\Command;
+use Illuminate\Hashing\BcryptHasher;
+
+class GeohubImport extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'geohub:import {--customer_email=*}'; // <-- allows to pass multiple values in the command line to retrieve tracks
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import geohub data. If no customer email is provided, all tracks and users will be imported.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void
+    {
+        //IMPORT USERS
+        $usersData = json_decode(file_get_contents('https://geohub.webmapp.it/api/export/editors'), true);
+        $this->importUsers($usersData);
+
+        $tracksData = json_decode(file_get_contents('https://geohub.webmapp.it/api/export/tracks'), true);
+        $this->importTracks($tracksData);
```

Comment:
In questo modo il fatto di selezionare un utente con il parametro customer_email non serve ad ottimizzare l'import infatti vegono scricati tutti i dati e poi filtrati dopo averli scaricati. Questo si potrebbe risolvere modifciando la API https://geohub.webmapp.it/api/export/tracks aggiungendo un parametro per filtrare:

https://geohub.webmapp.it/api/export/tracks/{customer_email} che restituisce lo stesso dato ma restituiendo solo le tracks che appartendono al customer_email

NOTA per Pedram: la API copmpleta restituisce "solo" 3200 (circa EcTracks.) Come mai?